app/operations/ops/idea/submit.rb

```diff
@@ -9,7 +9,7 @@ class Submit < BaseOperation
       step Contract::Build(constant: ::Propose::ProposeForm)
       step Contract::Validate()
       step Contract::Persist()
-      success :send_message_to_slack!
+      success :send_message_to_slack! unless Rails.env.development?
```

Comment:
That's a bad idea :( The code itself should now know anything about the specific environment, that we are using.
Can you explain, why do you want to disable this for development?

Reply:
Let me reproduce it on my machine, it should work even without this patch.

Reply:
https://monosnap.com/file/iq2siMzDj5LDk13VAOH8gFMjAvKgVS

Reply:
The reason is that you don't have redis running. It is required for running the application locally.

Reply:
so, probably this should be into documentation also